pyspecProcScripts/load_data.py

```diff
@@ -684,6 +691,7 @@ def proc_spincore_generalproc_v1(s, fl=None):
         #            this
     # {{{ always put the phase cycling dimensions on the outside
     neworder = [j for j in s.dimlabels if j.startswith("ph")]
+    neworder.sort() # sort w/ ph1 on outside ...
```

Comment:
why can't this use the generic processing routine and then only do the extra stuff? (Just look at the other functions where I've used it)

Reply:
this is the generic processing routine